src/plugins/simulator/fscommon/fsuipcimpl.cpp

```diff
@@ -148,10 +148,10 @@ namespace BlackSimPlugin::FsCommon
         Q_ASSERT_X(CThreadUtils::isInThisThread(this), Q_FUNC_INFO, "Open not threadsafe");
         if (!this->isOpened()) { return false; }
 
-        quint16 com1ActiveRaw = static_cast<quint16>(aircraft.getCom1System().getFrequencyActive().value(CFrequencyUnit::MHz()) * 100);
-        quint16 com2ActiveRaw = static_cast<quint16>(aircraft.getCom2System().getFrequencyActive().value(CFrequencyUnit::MHz()) * 100);
-        quint16 com1StandbyRaw = static_cast<quint16>(aircraft.getCom1System().getFrequencyStandby().value(CFrequencyUnit::MHz()) * 100);
-        quint16 com2StandbyRaw = static_cast<quint16>(aircraft.getCom2System().getFrequencyStandby().value(CFrequencyUnit::MHz()) * 100);
+        quint16 com1ActiveRaw = static_cast<quint16>(aircraft.getCom1System().getFrequencyActive().value(CFrequencyUnit::kHz()) / 10);
+        quint16 com2ActiveRaw = static_cast<quint16>(aircraft.getCom2System().getFrequencyActive().value(CFrequencyUnit::kHz()) / 10);
+        quint16 com1StandbyRaw = static_cast<quint16>(aircraft.getCom1System().getFrequencyStandby().value(CFrequencyUnit::kHz()) / 10);
+        quint16 com2StandbyRaw = static_cast<quint16>(aircraft.getCom2System().getFrequencyStandby().value(CFrequencyUnit::kHz()) / 10);
```

Comment:
The result of this conversion should be the same, right? What is the reason for this change?

Reply:
The old version with MHz causes rounding errors at 131.200 and 132.200 and then gives 131.19 and 132.19 respectively